docs/asset-store/docs/03-03-minio-gateway.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Minio and Minio Gateway
+type: Details
+---
+
+The whole concept of the Asset Store relies on Minio as a storage solution. It supports the Kyma's manifesto and the "batteries included" rule as Kyma provides you with this on-premise solution by default.
+
+Depending on the usage scenario, you can:
+- Use Minio for local development.
+- Store your assets on a production scale using Minio in a [Gateway mode](https://github.com/minio/minio/tree/master/docs/gateway).
+
+The Asset Store ensures that both usage scenarios work for Kyma, without additional configuration of the in-build controllers.
+
+## Local storage
+
+Minio is an open-source asset storage server with Amazon S3 compatible API. It can be used for storing various types of assets, such as documents, files, or images.
+
+In the context of the Asset Store, the Asset Controller stores all assets in Minio, in a dedicated storage space.
+
+![](assets/minio.svg)
+
+
+## Production storage
+
+For the production purposes, the Asset Store uses Minio Gateway which:
+
+- Is a multi-cloud solution that offers the flexibility to choose a given cloud provider for the specific Kyma installation, including Azure, Amazon, and Google
+- Supports data replication and CDN configuration
```

Comment:
small misunderstanding
Minio Gateway doesn supports data replication and CDN configuration, the cloud providers that we can use through the gateway support that

Reply:
ok, clear, thanks